### ISSUE TITLE
handleSelectItem not being called.

### ISSUE THREAD

**Issue by dooleyb1** [maxkordiyak/react-native-dropdown-autocomplete#29]
Hi there, 

The callback on selection doesn't seem to be firing for me. I've replaced the function with a simple `console.log` which is not being called also. Just wondering if you might be able to shed some light as to why? Thanks.

```javascript
<Autocomplete
    data={data}
    handleSelectItem={(item, index) => console.log('CAN YOU HEAR ME')}
    minimumCharactersCount={1}
    highlightText
    valueExtractor={item => item.name}
    rightContent
    rightTextExtractor={item => item.code}
    initialValue={this.props.college}
    placeholder="e.g Trinity College Dublin"
    spinnerColor="#E14ED2"
    highLightColor="#E14ED2"
    noDataText="Invalid College/University"
    inputStyle={styles.input}
    containerStyle={styles.dropdownContainer}
    scrollStyle={styles.dropdownContainer}
/>
```

**Comment by PaitoAnderson** [comment]
I had this issue too and resolved it in #28

**Comment by dooleyb1** [comment]
@PaitoAnderson Bang on, that fixed it. Thanks man 👍
